web/client/utils/LayersUtils.js

```diff
@@ -150,10 +150,42 @@ export const getNestedGroupTitle = (id, groups = []) => {
         return nodeObj ? nodeObj.title : null;
     }));
 };
+
+/**
+ * Flatten nested array to a one-level array
+ * @param {array} array of objects
```

Comment:
if you have to uses arrays of object as type you have to do like this
```suggestion
 * @param {object[]} array of objects
```